lib/games.rb

```diff
@@ -37,3 +39,4 @@ def all_events_summary
     end.join("\n")
   end
 end
+# binding.pry
```

Comment:
Be sure you're checking what you're committing. Did you expect there to be changes in this file? Probably not. A little alarm should have gone off when you ran `git add ./lib/games.rb`. You can also use `git diff` to see what changes you've made to a file. If they're not changes you want to commit, then either undo them or don't add the file to staging.